### PR TITLE
build!: use vrs-python for VRS 2.0-alpha models

### LINKED ISSUE #238: Replace metaschema with 2.0-alpha changes

### PR DESCRIPTION
**Pull request by korikuzma**
Close #238 . This work is needed for plenary. Our collaborators at the Broad will need to re-run the GK pilot data using variation-normalizer, which has gene as a dependency. They will be blocked until this is released.

Notes:
* Updates to VRS 2.0-alpha models provided by vrs-python 
  * VRSATILE descriptors are removed. Since we are trying to get something out before plenary, I just copied the models over from ga4gh.vrsatile.pydantic. Once we are in a good state for plenary, we can come back and revisit what we want to return in the response and cleanup.
* Removes ga4gh.vrsatile.pydantic dependency
* Temporarily removes adding chromosome location since it is not in the 2.0-alpha schema yet. It's waiting to be revisited. I just commented out the chromosome location parts. It feels bad to leave in a bunch of commented out code, so I can remove and just add back once we have the updated model. 
* some docs stuff was updated from replace all. Not focusing on docs since we're just trying to get this out. I can make an issue to update the docs for 2.0-alpha
* Some SequenceLocation/ChromosomeLocation work is updated in #245

### CODE REVIEW

**Comment by korikuzma** [comment]
Will open once a dev release is made in vrs-python

File: Pipfile [hunk 1]
```diff
@@ -9,8 +9,7 @@ fastapi = "*"
 uvicorn = "*"
 click = "*"
 boto3 = "*"
-"ga4gh.vrsatile.pydantic" = ">=0.1.0.dev7"
-"ga4gh.vrs" = {version = ">=0.9.0.dev0"}
+"ga4gh.vrs" = {version = "~=2.0.0.dev0"}
```
Comment (jsstevenson) [inline Pipfile:12]: ‼️

Comment (korikuzma) [reply Pipfile:12]: Feels weird, right? We want the VRS and VRS-Python major versions to match

**Review by jsstevenson — APPROVED** [review]
👍 looks like remaining TODO's will be revisiting chromosome location and updating the VRS section of the documentation. I'm guessing the plan is to tackle them later in new isssues?

side note: from the way that we handle VRS objects in the database, I'm wondering if dumps from gene normalizer builds based on VRS 1.x will be incompatible with metaschema/2.x builds (I think so?)

**Comment by korikuzma** [comment]
> 👍 looks like remaining TODO's will be revisiting chromosome location and updating the VRS section of the documentation. I'm guessing the plan is to tackle them later in new isssues?
> 
> side note: from the way that we handle VRS objects in the database, I'm wondering if dumps from gene normalizer builds based on VRS 1.x will be incompatible with metaschema/2.x builds (I think so?)

Right, I can make a new issue now for CL. 

Nope, gene-normalizer dumps are good! We create the VRS objects on the fly in the query handler

**Pull request merged**